include/zenoh-pico/api/primitives.h

```diff
@@ -52,21 +53,46 @@ z_string_t z_string_make(const char *value);
  */
 z_keyexpr_t z_keyexpr(const char *name);
 
+/**
+ * Constructs a :c:type:`z_keyexpr_t` departing from a string.
+ * It is a loaned key expression that aliases ``name``.
+ * Equal to z_zeyxepr: neither of them tests passed expression to correctness
```

Comment:
I do not get the point on this comment. Can you clarify?
Also, cite other functions using :c:func:\`z_zeyxepr\` block.

Reply:
In the zenoh-c the functions z_keyexpr differs from z_keyexpr_unchecked: the first one validates passed expression, senond one - don't. I wanted to point to the fact that the in the zenoh-pico validation is not performed in both functions. Will clarify the comment